vsce/server/src/server.ts

```diff
@@ -337,7 +335,7 @@ async function validateTextDocument(textDocument: TextDocument): Promise<void> {
 						`${err.errorMessage}`,
 						'Reach compilation encountered an error.',
 						DiagnosticSeverity.Error,
-						DIAGNOSTIC_TYPE_COMPILE_ERROR,
+						err.code,
```

Comment:
I'm accepting this. You should also generate a link

Reply:
To do that, we need to update the dependencies, which I've already done locally, and will now push shortly.

Reply:
Update: #410 does this.